client/state/selectors/get-billing-transaction-filters.js

```diff
@@ -0,0 +1,23 @@
+/** @format */
+/**
+ * External dependencies
+ */
+import { get } from 'lodash';
+
+/**
+ * Returns filter for the billing transactions of the given type
+ *
+ * @param  {Object}  state           Global state tree
+ * @param  {String}  transactionType Transaction type
+ * @return {Object}                 Billing transaction filters
+ */
+export default ( state, transactionType ) => {
+	const filters = get( state, [ 'ui', 'billingTransactions', transactionType ], {} );
+	return {
+		app: '',
```

Comment:
Do we really need the default values here? Could we rely on the initial reducer values instead? Would require some changes to the reducers though to make sure we default to the right values.